conf/modules.config

```diff
@@ -392,6 +392,35 @@ if (params.source == "fastq" | params.source == "genome_bam") {
 // DEXSeq Exon options
 //
 
+if (params.source == "fastq" | params.source == "genome_bam") {
+
+    if (params.leafcutter) {
+
+        process {
+
+            withName: 'REGTOOLS_JUNCTIONSEXTRACT' {
+                ext.args   = '-a 8 -i 50 -I 500000'
```

Comment:
are there long names for these arguments available? If so, please use them instead to make it more readable.

Reply:
no long parameters for this. I will add some comments